lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2335,6 +2336,36 @@ SwiftLanguageRuntime::GetRuntimeUnwindPlan(ProcessSP process_sp,
     return UnwindPlanSP();
   }
 
+  // If we're in the prologue of a function, don't provide a Swift async
+  // unwind plan.  We can be tricked by unmodified caller-registers that
+  // make this look like an async frame when this is a standard ABI function
+  // call, and the parent is the async frame.
+  // This assumes that the frame pointer register will be modified in the
+  // prologue.
+  Address pc;
+  pc.SetLoadAddress(regctx->GetPC(), &target);
+  SymbolContext sc;
+  if (pc.IsValid() && pc.CalculateSymbolContext(
+                          &sc, eSymbolContextFunction | eSymbolContextSymbol)) {
```

Comment:
anyone know off hand when this would return one of {function,symbol}, but not the other?